.github/workflows/kitchen.yml

```diff
@@ -137,19 +92,13 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest]
-        ruby: ["3.1"] # macos-11.0 is not public for now
+        ruby: ['3.1']
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out code
         uses: actions/checkout@v3
         with:
             clean: true
-      - name: 'Setup Ruby'
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: "3.1"
-          bundler-cache: false
```

Comment:
This needed to be set to true so as to run `bundle install` itself
https://github.com/ruby/setup-ruby#single-job